oracles.asciidoc

```diff
@@ -1,8 +1,8 @@
 == Oracles
 
-A key property of the Ethereum Virtual Machine is its ability to execute smart contract bytecode in a fully deterministic fashion. The EVM guarantees that the same operations will return identical outputs, regardless of the computer upon which they were actually run. While this characteristic is key to Ethereum's security assurances, the constraints of executing smart contracts within this deterministically walled garden impose strict limitations on their functionality, since the retrieval and processing of real-world data from off-chain is prevented. 
+A key property of the Ethereum Virtual Machine is its ability to execute smart contract bytecode in a fully deterministic fashion. The EVM guarantees that the same operations will return identical outputs, regardless of the computer upon which they were actually run. While this characteristic is key to Ethereum's security assurances, the constraints of executing smart contracts within this deterministically walled garden impose strict limitations on their functionality since the retrieval and processing of real-world data from off-chain are prevented. 
 
-There are a wide range of applications where smart contract might require such extrinsic information, and it is here that *oracles* come into play. Oracles can be defined as authoritative sources of off-chain data which allow smart contracts to receive and condition their execution using this information - they can be thought of as a mechanism for bridging the gap between on and off-chain. Extending the scope of smart contracts to enforce contractual relationship based on real-world events and data broadens their scope dramatically. Examples of data that might be provided by oracles include:
+There is a wide range of applications where the smart contract might require such extrinsic information, and it is here that *oracles* come into play. Oracles can be defined as authoritative sources of off-chain data which allow smart contracts to receive and condition their execution using this information - they can be thought of as a mechanism for bridging the gap between on and off-chain. Extending the scope of smart contracts to enforce contractual relationship based on real-world events and data broadens their scope dramatically. Examples of data that might be provided by oracles include:
```

Comment:
I actually think the plural use of 'range' (with 'are') is more prevalent - 'is' here sounds strange to me!

https://english.stackexchange.com/questions/19399/is-a-wide-range-of-features-singular-or-plural